main.js

```diff
@@ -175,8 +222,9 @@ define(function (require, exports, module) {
         }
     }
 
-    function onCheckedStateChange() {
-        _preferences.set("checked", Boolean(_command.getChecked()));
+    function onCheckedStateChange(e) {
+        _preferences.set("checked", e.target._checked);
```

Comment:
This is what happens when you do late night coding. :stuck_out_tongue: Found the real cause and reverted this.